apps/app/src/test/scala/com/daml/network/integration/tests/ManualStartIntegrationTest.scala

```diff
@@ -1,195 +0,0 @@
-package com.daml.network.integration.tests
```

Comment:
As discussed, this is expected: The 0.1.18 should have deleted those files already but we borked it